File: Golang/percabangan/konversi-nilai-angka-ke-huruf/konversi-nilai-angka-ke-huruf.go
// program konversi nilai angka yang diinput user menjadi huruf
package main

import "fmt"

func main() {
	var nilai int

	fmt.Print("Masukkan nilai kamu: ")
	fmt.Scan(&nilai)

	if nilai <= 100 && nilai >= 85 {
		fmt.Printf("Nilai kamu %d, kamu mendapat A.\n", nilai)
	} else if nilai >= 70 {
		fmt.Printf("Nilai kamu %d, kamu mendapat B.\n", nilai)
	} else if nilai >= 40 {
		fmt.Printf("Nilai kamu %d, kamu mendapat C.\n", nilai)
	} else if nilai >= 20 {
		fmt.Printf("Nilai kamu %d, kamu mendapat D.\n", nilai)
	} else {
		fmt.Printf("Nilai kamu %d, kamu mendapat E.\n", nilai)
	}
}

File: C++/konversi-nilai-angka-ke-huruf.cpp
#include <iostream>

using namespace std;

int main()
{
    int nilai;

    cout << "Masukkan nilai kamu: ";
    cin >> nilai;

    if (nilai <= 100 && nilai >= 85)
    {
        cout << "Nilai kamu " << nilai << ", kamu mendapat A.\n";
    }
    else if (nilai >= 70)
    {
        cout << "Nilai kamu " << nilai << ", kamu mendapat B.\n";
    }
    else if (nilai > 40)
    {
        cout << "Nilai kamu " << nilai << ", kamu mendapat C.\n";
    }
    else if (nilai >= 20)
    {
        cout << "Nilai kamu " << nilai << ", kamu mendapat D.\n";
    }
    else
    {
        cout << "Nilai kamu " << nilai << ", kamu mendapat D.\n";
    }
}
